GitUI/CommandsDialogs/FormCommit.Designer.cs

```diff
@@ -1178,11 +1180,25 @@ private void InitializeComponent()
             this.ResetAuthor.UseVisualStyleBackColor = true;
             this.ResetAuthor.Visible = false;
             // 
+            // StashStaged
```

Comment:
These controls should be in the bottom of the side.
Mono did not support that setup, why it was like this originally.

Reply:
Reverted to only insert `StashStaged`.
Have I got it right?
Should the `private Button ResetUnStaged;` be moved up to the section with the other `Button`s, too?